src/main/java/frc/robot/subsystems/Vision.java

```diff
@@ -0,0 +1,83 @@
+package frc.robot.subsystems;
+
+import com.spikes2212.command.DashboardedSubsystem;
+import com.spikes2212.util.Limelight;
+import edu.wpi.first.cameraserver.CameraServer;
+import edu.wpi.first.math.geometry.Pose3d;
+import org.photonvision.PhotonCamera;
+import org.photonvision.targeting.PhotonPipelineResult;
+
+public class Vision extends DashboardedSubsystem {
```

Comment:
`Vision` is not a subsystem and therfore should not extend `DashboardedSubsystem`. Just give it its own `RootNamespace` and don't forget to add an `update()` or `periodic()` method.